components/home/Home.jsx

```diff
@@ -1,10 +1,85 @@
 import * as React from 'react';
-import { Text, View } from 'react-native';
-
-export default (props) => {
-   return (    
-      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-         <Text>Home Screen</Text>
-         <Text>Replace these Text tags with your component</Text>
-      </View>);
-}
\ No newline at end of file
+import { Text, View, Button, StyleSheet } from 'react-native';
+import { ScreenStackHeaderCenterView } from 'react-native-screens';
+
+export default class App extends React.Component {
+
+   render() {
+      return (
+         <View style = {styles.container}>
+
+
+            {/*Cycles Section*/}
+            <View style = {[{width: "95%", height: "40%", margin: 30, backgroundColor: "F6F6F6", borderWidth: 5, borderRadius: 15, borderColor: "#5DB075"}]}>
```

Comment:
Which way of applying style to elements do you prefer? I think we should pick one so all of our code is the same. I personally prefer defining them in the styles object you have at the bottom, I think it makes the code more readable.

Reply:
I agree if we can figure out how to separate the shared style attributes and the unique attributes.

Reply:
Fixed